pkgs/by-name/fi/fichub-cli/package.nix

```diff
@@ -0,0 +1,60 @@
+{ lib, python3Packages, fetchPypi }:
+
+python3Packages.buildPythonApplication rec {
+  pname = "fichub-cli";
+  version = "0.10.2";
+  pyproject = true;
+
+  src = fetchPypi {
+    # Before version 0.10.0, we could `inherit pname` because the PyPI files were under the URL with dashes
+    # From version 0.10.0 onward, the files are only accessible via the URL with underscores
+    pname = "fichub_cli";
+    inherit version;
+    hash = "sha256-Cfux/guFYX6t/HnMtU06j3TCuRZBR9E1ZMYwORZTfnw=";
+  };
+
+  build-system = with python3Packages; [
+    setuptools
+  ];
+
+  dependencies = with python3Packages; [
+    platformdirs
+    beautifulsoup4
+    click
+    click-plugins
+    colorama
+    loguru
+    requests
+    tqdm
+    typer
+  ];
+
+  pythonImportsCheck = [
+    "fichub_cli"
+  ];
+
+  nativeCheckInputs = with python3Packages; [
+    pytestCheckHook
+  ];
+
+  # The package tries to create a file under the home directory on import
+  preCheck = ''
+    export HOME=$(mktemp -d)
+  '';
+
+  # These all attempt to access the network
+  disabledTests = [
+    "test_cli_url"
+    "test_cli_list_url"
+    "test_cli_infile"
+  ];
```

Comment:
```suggestion
  disabledTestPaths = [
    # Loading tests tries to download something from pypi.org
    "tests/test_cli.py"
  ];
```

Reply:
The reason I disabled those tests individually and not the entire file is that there is one test in that file that doesn't try to access the network and thus succeeds (even though all it is doing is checking the version). Did you mean to disable that as well?

Reply:
Never mind, I just got an error for this test as well. Not sure why it was working initially